static/src/javascripts-legacy/projects/common/modules/navigation/user-account.js

```diff
@@ -1,13 +1,15 @@
 define([
     'fastdom',
-    'common/modules/identity/api'
+    'common/modules/identity/api',
+    'lib/config',
 ], function (
     fastdom,
-    id
+    id,
+    config
 ) {
 
     function updateCommentLink () {
-        var commentLink = document.querySelector('.js-add-comment-activity-link');
+        var commentLink = document.querySelector('.js-add-comment-activity');
```

Comment:
Luckily this is already decoupled. The old header is managed by [`/projects/common/modules/navigation/profile.js`](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/modules/navigation/profile.js#L40).